app/js/components/LabResultEntry.jsx

```diff
@@ -236,160 +235,158 @@ export class LabResultEntry extends PureComponent {
 
     // This is a DOM tree containing all the form input elements
     const observations = (
-      <Grid fluid={true}>
```

Comment:
The removal of the outer `Grid` tag is the only change (use "hide whitespace changes" to review)